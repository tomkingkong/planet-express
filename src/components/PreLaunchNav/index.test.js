import React from 'react';
import { shallow } from 'enzyme';

import { PreLaunchNav } from '.';

describe('PreLaunchNav component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<PreLaunchNav />);
    expect(wrapper).toMatchSnapshot();
  });
});