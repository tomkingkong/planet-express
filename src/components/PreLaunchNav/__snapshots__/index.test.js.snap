// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreLaunchNav component should match snapshot 1`] = `
<div
  className="preLaunchNav"
>
  <Link
    className="launch"
    replace={false}
    to="/Launch"
  >
    <p>
      launch!
    </p>
  </Link>
  <Link
    className="news"
    replace={false}
    to="/News"
  >
    <p>
      news
    </p>
  </Link>
</div>
`;
